woocommerce/woocommerce
# Made JS to look for closest form-row paragraph instead of parent in the checkout.

As the input/select are now wrapped in span, going to parent brakes the form in some workflows.

### All Submissions:






### Changes proposed in this Pull Request:



Closes #19856 .

### How to test the changes in this Pull Request:

0. Checkout master.
1. Add a product into the cart.
2. Go to checkout.
3. Select Billing country without States, e.g. Denmark.
4. Refresh checkout page.
5. Change Billing country to United States.
6. Observe no State field is added.
7. Try to place order, get an error message "Billing State is a required field."
8. Switch to this branch.
9. Open checkout, select Billing country without States, e.g. Denmark.
10. Change Billing country to United States.
11. Observe State field is added.
12. Change Billing country to Cyprus.
13. Observe State changes to State/County.
14. Change Billing country to Denmark.
15. Observe State field disappears (as it should).

### Other information:

* [ ] Have you successfully ran tests with your changes locally? some e2e test are failing a bit randomly--different ones locally and one on travis, but does not seem to be related.. :/



### Changelog entry

> Fixed country selection form for checkout issue introduced by wrapping inputs fields in span.
